hack/common.sh

```diff
@@ -115,3 +113,18 @@ get_version() {
 
   echo $VERSION
 }
+
+# Given the parameter, adds operator image to csv or reverts the changes made to csv
+# Parameters:
+# 1: Optional parameter with operator image value to be replaced in the csv
+replace_image() {
```

Comment:
This can give rise to subtle issues if used incorrectly. Better is to make the function take in two inputs and specify the inputs i.e. `replace_image $OPERATOR_IMAGE REPLACE_IMAGE` vs `replace_image REPLACE_IMAGE $OPERATOR_IMAGE`. At which point the function can be `replace_csv_value` or something  along those lines.